src/components/PokemonList/PokemonList.tsx

```diff
@@ -0,0 +1,22 @@
+import { ErrorBox } from '@/components/ErrorBox/ErrorBox';
+import { Loader } from '@/components/Loader/Loader';
+import { usePokemons } from "@/hooks/usePokemons";
+import styles from '@/components/PokemonList/PokemonList.module.css';
+
+export const PokemonList = () => {
+  const { pokemons, errorArr, ignoreFetch } = usePokemons();
```

Comment:
```suggestion
  const { pokemons, errorArr, isLoading } = usePokemons();
```

or

```suggestion
  const { pokemons, errorArr, isFetching } = usePokemons();
```

Wouldn't this be more readable in this context when you're just using this hook?

```if (isFetching) { show loader }  ```
vs
```if (!ignoreFetch) { show loader } ```


The second one doesn't look that obvious to me, and it looks like the internals of usePokemons() are leaking outside.


WDYT?

Reply:
agree, much simpler.